### PR TITLE
1. unify letter case.

### PR DESCRIPTION
**Pull request by paul-cheung**
2. "collection of channels" to "collection of channel".

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[82e0766](https://github.com/paul-cheung/microsoft-graph-docs/pull/4039/commits/82e076627459f87d35e87a84102d53ffa2bed26d)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[82e0766](https://github.com/paul-cheung/microsoft-graph-docs/pull/4039/commits/82e076627459f87d35e87a84102d53ffa2bed26d)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[82e0766](https://github.com/paul-cheung/microsoft-graph-docs/pull/4039/commits/82e076627459f87d35e87a84102d53ffa2bed26d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/team.md](https://github.com/paul-cheung/microsoft-graph-docs/blob/letter_case/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902250938551232-4039%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Rx%2BJ67Kl5CdK29o0pnHY4ByaoXai6rBOg%2FRsk6%2BnLC0%3D&st=2019-02-25T09%3A37%3A17Z&se=2019-03-28T09%3A42%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[c9f2e73](https://github.com/paul-cheung/microsoft-graph-docs/pull/4039/commits/c9f2e739184ef101223b44db60536242b7bef2bb)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

File: api-reference/beta/resources/team.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ ms.prod: "microsoft-teams"
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-A team in Microsoft Teams is a collection of [channels](channel.md). 
+A team in Microsoft Teams is a collection of [channel](channel.md) objects.
```
Comment (Lauragra) [inline api-reference/beta/resources/team.md]: Please make the same edits to the v1.0 version of the same topic. Thanks.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[c9f2e73](https://github.com/paul-cheung/microsoft-graph-docs/pull/4039/commits/c9f2e739184ef101223b44db60536242b7bef2bb)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[c9f2e73](https://github.com/paul-cheung/microsoft-graph-docs/pull/4039/commits/c9f2e739184ef101223b44db60536242b7bef2bb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/team.md](https://github.com/paul-cheung/microsoft-graph-docs/blob/letter_case/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902251717286147-4039%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=QhEbMDWRcuYtsSAq4dBlhhY3YUIbQilj8nAmFhmDyPY%3D&st=2019-02-25T17%3A15%3A55Z&se=2019-03-28T17%3A20%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/resources/team.md (hunk 1 above)
Comment (paul-cheung) [reply api-reference/beta/resources/team.md]: ok

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a569594](https://github.com/paul-cheung/microsoft-graph-docs/pull/4039/commits/a56959492d3d7020d440b629f3057f45094dada2)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a569594](https://github.com/paul-cheung/microsoft-graph-docs/pull/4039/commits/a56959492d3d7020d440b629f3057f45094dada2)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a569594](https://github.com/paul-cheung/microsoft-graph-docs/pull/4039/commits/a56959492d3d7020d440b629f3057f45094dada2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/team.md](https://github.com/paul-cheung/microsoft-graph-docs/blob/letter_case/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/team.md](https://github.com/paul-cheung/microsoft-graph-docs/blob/letter_case/api-reference/v1.0/resources/team.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C26%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902260323412457-4039%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=yUczEGc6ps7izVCLsiSaRFjz4txXoZZvjh4zK8vrnjg%3D&st=2019-02-26T03%3A22%3A11Z&se=2019-03-29T03%3A27%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by paul-cheung** [comment]
could this commit be merged?

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[296e7a9](https://github.com/paul-cheung/microsoft-graph-docs/pull/4039/commits/296e7a9afbc49b80b6c8b17508fcbf529993de74)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/team.md](https://github.com/paul-cheung/microsoft-graph-docs/blob/letter_case/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/team.md](https://github.com/paul-cheung/microsoft-graph-docs/blob/letter_case/api-reference/v1.0/resources/team.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C28%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903281613271425-4039%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=a8zjTpBShCHtTRxadNGMQjQ1MHMk4M54OIjvf%2FCDimo%3D&st=2019-03-28T16%3A12%3A09Z&se=2019-04-28T16%3A17%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Lauragra** [comment]
Yes, thanks for making the update!

**Pull request merged**